### PR TITLE
Moved away from horstbaerbel's repo. Used ubuntu:rolling so that we g…

### PR DESCRIPTION
**Pull request by nolan-veed**
…et the latest clang-format-12 as ubuntu:latest doesn't have it.

# Why

All render-node devs seem to have IDEs setup to use clang-format-12.

So, inevitably, what happens is that IDEs format code differently compared to what the github action based on this repo expects. This can waste time not to mention become annoying and frustrating.

# What

The https://github.com/HorstBaerbel/ccpp-cmake-build-and-test docker image upon which https://github.com/HorstBaerbel/action-clang-format (and us) are based on still hasn't moved to clang-format-12. That is not available with ubuntu-latest, which is based on 20:04 (current LTS).

So, I've elected to use the ubuntu:rolling image which is every release os Ubuntu. We also don't need other tools, so those haven't been included.

See https://hub.docker.com/_/ubuntu

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,17 @@
-FROM horstbaerbel/ccpp-cmake-build-and-test:1.0
+FROM ubuntu:rolling
+
+WORKDIR /
+
+ENV TZ=Europe/Berlin
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```
Comment (jakubtyrcha) [inline Dockerfile:9]: I don't understand this. Why do we need `TZ=Europe/Berlin`? Is this the minimal amount of code to do the job?

Comment (akaltar) [inline Dockerfile:9]: Could you leave a comment on why is this needed please?

Comment (nolan-veed) [reply Dockerfile:9]: That's literally a copy-paste from the upstream repo.

I suspect, we need to tell ubuntu the machine is in. It doesn't matter too much I think.

Some application installs do require timezone to be setup... I think as we are doing a dist-upgrade and install, they might pull packages that require it.

I'm not sure.

Comment (nolan-veed) [reply Dockerfile:9]: I've seen this typically first thing when creating OS based images. Or locale even.

But, I'll add a clarification.

**Review by akaltar — APPROVED** [review]
lgtm
